cartographer/cloud/internal/client/pose_graph_stub.h

```diff
@@ -51,6 +51,7 @@ class PoseGraphStub : public ::cartographer::mapping::PoseGraphInterface {
   std::map<int, mapping::PoseGraphInterface::TrajectoryData> GetTrajectoryData()
       const override;
   std::vector<Constraint> constraints() const override;
+  std::chrono::milliseconds GetWorkQueueDelay() const override;
```

Comment:
Why limit the precision to milliseconds? We could use `std::chrono::duration<double>`. `cartographer/metrics` uses `double`, so it would fit very well should we decide to expose the work queue delay as a metric. This has the additional benefit of using an SI base unit which always seems a good idea.

@stevewolter @gaschler What do you think?

Reply:
I couldn't care less, I think. Both double and int millisecond precision are sufficient for all practical purposes.

Reply:
I prefer SI units and data types that we already use in the public API, like double in seconds or common::Duration, not milliseconds.

Reply:
Let's use seconds as double then, and add them directly to the metric system.

Reply:
Done.